src/components/KPrompt/KPrompt.vue

```diff
@@ -7,29 +7,28 @@
     :title="displayTitle"
   >
     <template #header-content>
-      <div class="k-prompt-header w-100">
-        <div class="k-prompt-header-content d-flex align-items-center w-100">
+      <div class="k-prompt-header">
+        <div class="k-prompt-header-content">
           <slot name="header-content">
             <KIcon
               v-if="type === 'warning'"
-              class="mr-2"
-              color="var(--white)"
+              class="warning-icon"
+              :color="`var(--white, ${KUI_COLOR_TEXT_INVERSE})`"
```

Comment:
Do we need the CSS variable token here? 
```suggestion
              :color="`var(--white, var(--kui-color-text-inverse, ${KUI_COLOR_TEXT_INVERSE}))`"
```

Reply:
Fixed